compiler/rustc_ast_pretty/src/pprust/state/expr.rs

```diff
@@ -244,6 +244,7 @@ impl<'a> State<'a> {
             (&ast::ExprKind::Let { .. }, _) if !parser::needs_par_as_let_scrutinee(prec) => {
                 parser::PREC_FORCE_PAREN
             }
+            (&ast::ExprKind::Match(..), _) => parser::PREC_FORCE_PAREN,
```

Comment:
Can you add a comment here like the other two cases which provides an example of why this is needed?

Reply:
Added.